openassessment/workflow/models.py

```diff
@@ -287,6 +287,17 @@ def update_from_assessments(self, assessment_requirements):
 
         step_for_name = {step.name: step for step in steps}
 
+        # If the status is done or cancelled, check if score has changed.
+        if self.status == self.STATUS.done:
+            if force_update_score:
+                new_score = self.get_score(assessment_requirements, step_for_name)
+                self.set_score(new_score)
+                self.save()
+                logger.info((
+                    u"Workflow for submission UUID {uuid} has updated score."
+                ).format(uuid=self.submission_uuid))
+            return
```

Comment:
I actually like the idea of a force option to the function. Though I wonder if it could be simplified, here, to just:
if self.status == self.STATUS.done and not force_update_score:
    return

If you did that, then you'd just go through the flow, per usual. That would more truly force the update of the workflow, and work the same way. If we ever wanted to use the 'force_update' override, it could be more repurposed.